lib/winnow.rb

```diff
@@ -22,14 +22,15 @@ def self.base_scope_method
   end
 
   def self.choose_base_scope_method
-    if rails_four?
+    if rails_has_all?
       :all
     else
       :scoped
     end
   end
 
-  def self.rails_four?
-    defined?(ActiveRecord::VERSION::STRING) && ActiveRecord::VERSION::STRING =~ /^4/
+  def self.rails_has_all?
+    defined?(ActiveRecord::VERSION::STRING) && 
+      (ActiveRecord::VERSION::STRING =~ /^4/  || ActiveRecord::VERSION::STRING =~ /^5/)
```

Comment:
You could use [Gem::Verison](https://docs.ruby-lang.org/en/2.5.0/Gem/Version.html) for this:

```suggestion
      Gem::Version.new(ActiveRecord::VERSION::STRING) >= Gem::Version.new(4)
```

Reply:
Thanks for spotting this. 

Looks like this check is legacy as we are only supporting Rails 4+ so we can just use `all` everywhere and avoid the need to check these versions.